helmclient.go

```diff
@@ -146,32 +136,7 @@ func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options
 }
 
 func (c *Client) deleteRelease(ctx context.Context, releaseName string, options ...helmclient.DeleteOption) error {
-	o := func() error {
-		t, err := c.newTunnel()
-		if IsTillerNotFound(err) {
-			return backoff.Permanent(microerror.Mask(err))
-		} else if err != nil {
-			return microerror.Mask(err)
-		}
-		defer c.closeTunnel(ctx, t)
-
-		_, err = c.newHelmClientFromTunnel(t).DeleteRelease(releaseName, options...)
-		if IsReleaseNotFound(err) {
-			return backoff.Permanent(microerror.Mask(err))
-		} else if err != nil {
-			return microerror.Mask(err)
-		}
-
-		return nil
-	}
-	b := backoff.NewMaxRetries(10, 5*time.Second)
-	n := backoff.NewNotifier(c.logger, ctx)
-
-	err := backoff.RetryNotify(o, b, n)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
+	c.logger.LogCtx(ctx, "level", "debug", "message", "delete release not yet implemented for helm 3")
```

Comment:
Helm 2 implementation logic is removed. It will be replaced with the new Helm 3 logic.